src/routes/safe/components/Balances/FetchTokens/index.jsx

```diff
@@ -0,0 +1,30 @@
+// @flow
+import { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+
+import fetchCollectibles from '~/logic/collectibles/store/actions/fetchCollectibles'
+import { fetchCurrencyValues } from '~/logic/currencyValues/store/actions/fetchCurrencyValues'
+import activateAssetsByBalance from '~/logic/tokens/store/actions/activateAssetsByBalance'
+import activateTokensByBalance from '~/logic/tokens/store/actions/activateTokensByBalance'
+import { fetchTokens } from '~/logic/tokens/store/actions/fetchTokens'
+import { extendedSafeTokensSelector } from '~/routes/safe/container/selector'
+import fetchTokenBalances from '~/routes/safe/store/actions/fetchTokenBalances'
+import { safeParamAddressFromStateSelector } from '~/routes/safe/store/selectors'
+
+const FetchTokens = () => {
```

Comment:
why pascal case? also, can this be a hook?

Reply:
fixed

Reply:
Removed pascal case, now it's a hook